rust/otel-arrow-rust/src/encode/record/logs.rs

```diff
@@ -591,6 +600,15 @@ impl ResourceBuilder {
         }
     }
 
+    /// Append the value of `id` to the builder `n` times
+    pub fn append_id_n(&mut self, val: Option<u16>, n: usize) {
```

Comment:
nit: Update the method signature to accept a `u16` instead of `Option<u16>`. I believe we would never call this method for `None` so we need not have that `if` check.

```suggestion
    pub fn append_id_n(&mut self, val: u16, n: usize) {
```

Reply:
Good call, made this change